types.d.ts

```diff
@@ -1,23 +1,31 @@
 import { Boolean, Float, Integer, String, True } from "./alias";
 
 declare namespace Update {
+  /** Internal type holding properties that updates in channels share. */
   interface Channel {
+    chat: { type: Chat.ChannelChat["type"] };
     author_signature?: String;
-    chat: { type: "channel" };
-    from?: never;
+    from: never;
   }
+  /** Internal type holding properties that updates outside of channels share. */
   interface NonChannel {
-    author_signature?: never;
+    chat: { type: Exclude<Chat["type"], Update.Channel["chat"]["type"]> };
+    author_signature: never;
     from: User;
   }
+  /** Internal type holding properties that updates about new messages share. */
+  interface New {
+    edit_date: never;
+  }
+  /** Internal type holding properties that updates about edited messages share. */
   interface Edited {
     edit_date: Integer;
-    forward_from?: never;
-    forward_from_chat?: never;
-    forward_from_message_id?: never;
-    forward_signature?: never;
-    forward_sender_name?: never;
-    forward_date?: never;
+    forward_from: never;
+    forward_from_chat: never;
+    forward_from_message_id: never;
+    forward_signature: never;
+    forward_sender_name: never;
+    forward_date: never;
```

Comment:
It's impossible to create a value of type `never`, so this `interface` is impossible to instantiate!
```suggestion
    forward_from?: never;
    forward_from_chat?: never;
    forward_from_message_id?: never;
    forward_signature?: never;
    forward_sender_name?: never;
    forward_date?: never;
```